### PR TITLE
Remove graduated GitHub v3 API preview header

### PR DESCRIPTION
**Pull request by n1lesh**
removed Accept header (with value mediaTypeCheckRunsPreview). Since checks run is out of preview, didn't see any value in keeping it.

Followed as per,

Graduation of preview: https://developer.github.com/changes/2020-10-01-graduate-antiope-preview/
Docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/checks

### CODE REVIEW

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @n1lesh , but this is the wrong approach.
We need to literally remove it, not replace it.

Please see https://github.com/google/go-github/pull/1618/files as a recent example of how this is done.

**Review by gmlewis — APPROVED** [review]
Perfect!  Thank you, @n1lesh !
LGTM.

Awaiting second LGTM before merging.

**Review by nightlark — APPROVED** [review]
LGTM

**Comment by gmlewis** [comment]
Thank you, @nightlark !
Merging.

**Pull request merged**
